src/config/index.ts

```diff
@@ -95,3 +95,29 @@ export const buildSafeCreationTxUrl = (safeAddress: string) => {
 
   return `${host}${base}`
 }
+
+export type BlockScanInfo = () => {
+  alt: string
+  url: string
+}
+
+export const getExplorerInfo = (hash?: string | null): BlockScanInfo => {
+  const { name, url } = getNetworkExplorerInfo()
+
+  const blockScanInfo = () => {
+    const result = {
+      url: '',
+      alt: '',
+    }
+    if (!hash) {
```

Comment:
In which cases would we need the use case of not making hash mandatory? Does it make sense to return empty strings?

Checking where we use this function  I see that we may only pass a null/undefined hash in this lines:
https://github.com/gnosis/safe-react/blob/649f7b3072fb34d065a513a8a8ce87d3c4a054dc/src/routes/safe/components/Transactions/TxsTable/ExpandedTx/CreationTx/index.tsx#L34

Wouldn't the problem be solved moving those 3 lines below the `if(!tx)` check that it's already in CreationTx?